scalding-core/src/main/scala/com/twitter/scalding/Config.scala

```diff
@@ -142,7 +165,11 @@ trait Config {
       case Left((bootstrap, inst)) => ConfiguredInstantiator.setSerialized(chillConf, bootstrap, inst)
       case Right(refl) => ConfiguredInstantiator.setReflect(chillConf, refl)
     }
-    Config(chillConf.toMap + hadoopKV)
+    val withKryo = Config(chillConf.toMap + hadoopKV)
```

Comment:
Yep, i just realised this myself after staring at some failing hadoop jobs for too long :/ Good catch